mimic/model/clb_objects.py

```diff
@@ -19,11 +19,15 @@ def _prep_for_list(lb_list):
     Removes tenant id and changes the nodes list to 'nodeCount' set to the
     number of node on the LB
     """
-    entries_to_keep = ('name', 'protocol', 'id', 'port', 'algorithm', 'status', 'timeout',
-                       'created', 'virtualIps', 'updated', 'nodeCount')
+    entries_to_keep = (
```

Comment:
For future PRs, I'd appreciate it if you could open a separate PR for re-factorings / re-formattings like this.  Since I'm looking at this as changes to CLBs, I assumed that this list had changed for some reason, and that the list comprehension below had also changed somehow, so I had to carefully read it repeatedly to make sure that I wasn't missing something.  If I were reviewing this in the context of a simple reformatting PR, I would just have scanned it to make sure that no elements in the list were different, and quickly hit the merge button.  So context is important and apparently trivial cleanups like this can slow down reviews a bit.

Please keep in mind that this suggestion comes from someone who does this *all the time*, so I'm noticing it because it's a bad habit I have :).